### PR TITLE
Reformat all code: with a line limit of 100.

### PR DESCRIPTION
**Pull request by kurtwheeler**
## Issue Number

#2022

## Purpose/Implementation Notes

This is a remake of #2022, run with a line limit of 100 instead of black's default 88. This is because we've been using a line limit of 100 for a while so a lot of code was written to fit neatly within 100 characters, so black had broken a lot of these up in unfortunate ways.

Also this shaves ~1600 lines of code out of our project compared to the 88 character line limit.

## Types of changes

What types of changes does your code introduce?

- Reformat

## Checklist

- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by kurtwheeler** [comment]
@arielsvn I added some manual changes to the README, install_all, and I added a script for us to copy over to the pre-commit hook. Wanna take a look? (I'm not sure the way I'm installing black on linux is general enough, should it sometimes be pip instead of just pip3?)

File: README.md [hunk 1]
```diff
@@ -122,11 +126,12 @@ The following services will need to be installed:
 - [git-crypt](https://www.agwa.name/projects/git-crypt/)
 - [iproute2mac](https://github.com/brona/iproute2mac)
 - [jq](https://stedolan.github.io/jq/)
+- [black](https://black.readthedocs.io/en/stable/)
 
 Instructions for installing [Docker](https://www.docker.com/docker-mac) and [Homebrew](https://brew.sh/) can be found by
 on their respective homepages.
 
-Once Homebrew is installed, the other required applications can be installed by running: `brew install iproute2mac git-crypt nomad terraform jq`.
+Once Homebrew is installed, the other required applications can be installed by running: `brew install iproute2mac git-crypt nomad terraform jq black`.
```
Comment (arielsvn) [inline README.md]: I installed `black` using `pip3` on Mac. Also tested it, and it doesn't work with `brew`.

File: scripts/install_all.sh [hunk 2]
```diff
@@ -152,6 +152,18 @@ if ! command -v nomad > /dev/null; then
     fi
 fi
 
+if ! command -v black > /dev/null; then
+    echo "Installing black..."
+    if [ $BREW ]; then
+        $INSTALL_CMD black > $OUTPUT
```
Comment (arielsvn) [inline scripts/install_all.sh]: should we always use `pip3` for `black`?

Comment (kurtwheeler) [reply scripts/install_all.sh]: Ok thanks for testing that!! I've made that change.

**Comment by kurtwheeler** [comment]
Ok, I just tested using `install_all.sh` to configure the pre-commit hook and then made a commit and it worked!!!

File: README.md (hunk 1 above)
Comment (arielsvn) [reply README.md]: We should remove black from here too.

```suggestion
Once Homebrew is installed, the other required applications can be installed by running: `brew install iproute2mac git-crypt nomad terraform jq`.
```

**Comment by arielsvn** [comment]
Ups my bad, seems that black can be installed with `brew`

**Pull request merged**